src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerGenotypingEngine.java

```diff
@@ -556,19 +557,11 @@ private VariantContext addNonRefSymbolicAllele(final VariantContext mergedVC) {
                                                                       final byte[] ref,
                                                                       final SimpleInterval refLoc,
                                                                       final List<VariantContext> activeAllelesToGenotype) {
-        final boolean in_GGA_mode = !activeAllelesToGenotype.isEmpty();
+        //TODO: maintaining previous behavior, which cleared the TreeSet from buildEventMaps before adding only the GGA positions
+        //TODO: This seems wrong -- GGA mode shouldn't discard non-GGA loci, right?
```

Comment:
I see. I was confused by `FeatureContext` vs `FeatureInput` but now it makes sense. Thanks!